openedx/core/djangoapps/user_api/accounts/tests/test_retirement_views.py

```diff
@@ -170,26 +172,12 @@ def test_on_jwt_headers_rejection(self):
 
 
 @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Account APIs are only supported in LMS')
-class TestDeactivateLogout(RetirementTestCase):
+class TestDeactivateLogout(RetirementTestCase, RetirementTestUserMixin):
```

Comment:
This is to avoid having mixins inherit from `TestCase`. This is related to [Python method resolution order](https://www.geeksforgeeks.org/method-resolution-order-in-python-inheritance/).

```suggestion
class TestDeactivateLogout(RetirementTestUserMixin, RetirementTestCase):
```

Reply:
Done